src/components/Stats/TopProjects/Project/Project.js

```diff
@@ -5,13 +5,13 @@ import "./Project.css"
 const Project = ({ title, description, stars, url }) => {
   return (
     <article className="project">
-      <div>
+      <div className="project__text">
         <a href={url}>
           <div className="project__title">{title}</div>
         </a>
         <div className="project__description">{description}</div>
       </div>
-      <div className="project__stars">{stars}</div>
+      <div className="project__stars">{stars}</div>&nbsp;⭐️
```

Comment:
The visual aspect while depend of OS's support of this unicode character, no ? I would recommend using a image in svg so the visual aspect is consistent for everyone ? This is also better for accessibility we can add an aria-label on the image to describe it 😄

Reply:
Done ! Is it worth describing an image when it brings no new information? Wouldn't an `alt=""` suffice?

Reply:
I found a [recommandation](https://stackoverflow.com/questions/4697100/accessibility-recommended-alt-text-convention-for-svg-and-mathml) to use `role=img` with and `aria-label` for svg tags

Reply:
Should I then directly use the `svg` tag instead of an `img`? I could create a component that wraps our `svg`

Reply:
It would be nice !

Reply:
Done 😄